ci/pr

```diff
@@ -5,6 +5,8 @@ set -e -u -x
 export GOPATH=$PWD/gopath
 export PATH=$PWD/gopath/bin:$PATH
 
+git clone git://github.com/SHyx0rmZ/baggageclaim -b async-volume-creation $GOPATH/src/github.com/concourse/baggageclaim
+
```

Comment:
Can we keep this until a possible merge or close happens? I added it so the PR pipeline would not complain about the BaggageClaim changes missing.